ldap/servers/plugins/replication/cl5_config.c

```diff
@@ -314,6 +314,12 @@ cldb_config_modify(Slapi_PBlock *pb, Slapi_Entry *e, Slapi_Entry *entryAfter, in
                     /* Storing the encryption symmetric key */
                     /* no need to change any changelog configuration */
                     goto done;
+
+
+                } else if (strcasecmp(config_attr, CONFIG_CHANGELOG_ENCRYPTION_ALGORITHM) == 0) {
+                    /* We should allow the operation to succeed but it requires
+                     * a restart to take effect. */
```

Comment:
No it is not.  So what this does, and the comment above about the "goto", is so we can set this over LDAP without it being rejected.  But it requires a server restart to taker effect.  Previously any attempt to change the config while the server was running resulted in an err 53.  Meaning you were forced to edit the dse.ldif while the server was stopped.  Now you can set it via ldapmodfy, and it won't take effect until the server is restarted.

Reply:
Okay. So it would probably be the same for CONFIG_CHANGELOG_SYMMETRIC_KEY (few lines above). Allow ldapmodify update  but do not set it in config.symmetricKey until the next restart.

Reply:
Would a user set this attribute?   I thought the "server" was the only thing that set this attribute.  If a user would add this value then I can add the exception

Reply:
> Okay. So it would probably be the same for CONFIG_CHANGELOG_SYMMETRIC_KEY (few lines above). Allow ldapmodify update but do not set it in config.symmetricKey until the next restart.

Looks like this was always allowed.  So I'm going to leave it as it was.